deltachat-ios/Controller/SettingsVideoChatViewController.swift

```diff
@@ -1,53 +1,156 @@
 import UIKit
 import DcCore
+
+class PredefinedVideoChatOptionCell: UITableViewCell {
+
+    public var url: String
+
+    init(label: String, url: String) {
+        self.url = url
+        super.init(style: UITableViewCell.CellStyle.subtitle, reuseIdentifier: label)
+        self.textLabel?.text = label
+        self.detailTextLabel?.text = url
+        self.detailTextLabel?.textColor = .lightGray
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+
 class SettingsVideoChatViewController: UITableViewController {
 
     private var dcContext: DcContext
 
-    private lazy var videoInstanceCell: TextFieldCell = {
-        let cell = TextFieldCell.makeConfigCell(labelID: String.localized("videochat_instance"),
+    private let defaultOptions = [
+        PredefinedVideoChatOptionCell(label: "Jitsi", url: "https://meet.jit.si/$ROOM"),
+        PredefinedVideoChatOptionCell(label: "Systemli", url: "https://meet.systemli.org/$ROOM"),
+        PredefinedVideoChatOptionCell(label: "Autistici", url: "https://vc.autistici.org/$ROOM"),
+    ]
+
+    private lazy var offCell: UITableViewCell = {
+        let cell = UITableViewCell(style: UITableViewCell.CellStyle.default, reuseIdentifier: "off")
+        cell.textLabel?.text = String.localized("off")
+        return cell
+    }()
+
+    private lazy var customInstanceCell: TextFieldCell = {
+        let cell = TextFieldCell.makeConfigCell(labelID: String.localized("custom"),
                                                 placeholderID: String.localized("videochat_instance_placeholder"))
         cell.textField.autocapitalizationType = .none
         cell.textField.autocorrectionType = .no
         cell.textField.textContentType = .URL
+        cell.textField.addTarget(self, action: #selector(setCustom), for: .editingDidBegin)
```

Comment:
why this one? It will save the existing settings again, before it was changed. I doubt this is necessary

Reply:
it makes sure that the checkmark is moved to "custom" when the text field is tapped. otherwise, you have to tap a second time on the label or so which is not so intuitive.